index.bs

```diff
@@ -514,12 +514,30 @@ To <dfn>handle an incoming message</dfn> given a [=WebSocket connection=]
 
 <div algorithm>
 To <dfn>respond with an error</dfn> given a [=WebSocket connection=]
-|connection| and an [=error code=] |code|:
+|connection|, |command id| and |error code|:
 
- 1. Issue: Form a valid JSON |errorObject| given |code|.
+ 1. If |command id| is not a number, let |command id| be null.
 
- 2. [=Send a WebSocket message=] comprised of |errorObject| over
-    |connection|.
+ 1. Let |error data| be a new map matching the <code>Error</code> production in
```

Comment:
Is it expected that every step has `1.` assigned to it? Maybe it will be correctly displayed in the rendered page?

Reply:
Yep, Markdown will render it with the correct numbers.

Reply:
Could we then stay in sync with the rest of the spec. Under `incoming message` the numbering is incremented for each step.

Reply:
Yeah, I'm slowly converting everything to the `1.` style. We will end up with something consistent, but I don't want to add that to this PR (could be a good candidate for a small editorial PR though)